python/lsst/summit/testing/analysis/m1m3/plots/inertia_compensation_system.py

```diff
@@ -0,0 +1,277 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+from astropy.time import Time
+
+# This import is for type hinting only and should not be used in runtime code.
+from lsst.summit.testing.analysis.m1m3.inertia_compensation_system import ICSAnalysis
```

Comment:
Does that mean it needs removing from here before merging?

Reply:
To be sincere, I don't know how to solve this. I was afraid that having the import here would cause cyclic imports. Do you know another approach?

Reply:
Sorry, if this is about type annotations then I'm afraid I know nothing - I've not gone down the mypy route at all. There are lots of knowledgeable people on the project though, so if this is about typing and mypy then if you ask in #software-dev you'll get plenty of help. I suspect it'll have something to do with "stubs" or something like that, but I know nothing about any of that stuff I'm afraid.

Reply:
You could use a `Protocol` to define what properties/method of `ICSAnalysis` you use. See my [other comment](https://github.com/lsst-sitcom/summit_testing_analysis/pull/2#discussion_r1339219122)